miden-crypto/Cargo.toml

```diff
@@ -62,11 +62,11 @@ harness = false
 name    = "rand"
 
 [features]
-concurrent = ["dep:rayon", "hashbrown?/rayon"]
+concurrent = ["dep:rayon"]
 default = ["concurrent", "std"]
 executable = ["dep:clap", "dep:rand-utils", "std"]
 fuzzing = []
-hashmaps = ["dep:hashbrown"]
+hashmaps = ["std"]
```

Comment:
I agree with @krushimir - I don't think there is too much value in having a separate `hashmaps` feature because we can just use `std` feature to switch between `HashMap` and `BTreeMap`.